upgrade_guide/migrating_from_astyanax/configuration/README.md

```diff
@@ -0,0 +1,242 @@
+# Configuration
+
+## How Configuring the Java driver works
+
+The two basic components in the Java driver are the `Cluster` and the `Session`.
+The `Cluster` is the object to create first, and on to which apply all
+global configuration options. Connecting to the `Cluster` creates a
+`Session`. Queries are executed through the `Session`.
+
+The `Cluster` object then is to be viewed as the equivalent of the `AstyanaxContext`
+object. ’Starting’ an `AstyanaxContext` object typically returns a `Keyspace`
```

Comment:
\[typo maniac warning] Surrounding "Starting" with two closing quotes looks ugly. If you use plain quotes in the source (`'Starting'`), documentor will do the right thing and generate an opening and closing quote. This works with double quotes as well.